2025-07-01 07:10:16,909 - INFO - [Rank 0] Initialized wandb at local_rank 0
Traceback (most recent call last):
  File "/workspace/HLLM/main/train.py", line 29, in <module>
    trainer.train()
  File "/workspace/HLLM/main/train_utils.py", line 186, in train
    rank = dist.get_rank()
          ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/torch/distributed/distributed_c10d.py", line 2297, in get_rank
    default_pg = _get_default_group()
                 ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/torch/distributed/distributed_c10d.py", line 1298, in _get_default_group
    raise ValueError(
ValueError: Default process group has not been initialized, please make sure to call init_process_group.
[rank0]: Traceback (most recent call last):
[rank0]:   File "/workspace/HLLM/main/train.py", line 29, in <module>
[rank0]:     trainer.train()
[rank0]:   File "/workspace/HLLM/main/train_utils.py", line 186, in train
[rank0]:     rank = dist.get_rank()
[rank0]:            ^^^^^^^^^^^^^^^
[rank0]:   File "/usr/local/lib/python3.11/dist-packages/torch/distributed/distributed_c10d.py", line 2297, in get_rank
[rank0]:     default_pg = _get_default_group()
[rank0]:                  ^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/usr/local/lib/python3.11/dist-packages/torch/distributed/distributed_c10d.py", line 1298, in _get_default_group
[rank0]:     raise ValueError(
[rank0]: ValueError: Default process group has not been initialized, please make sure to call init_process_group.
